### ISSUE TITLE
Multiple patterns for TestContainText

### ISSUE THREAD

**Issue by DronMDF** [DronMDF/2out#295]
Like this:
```c++
        make_shared<TestContainText>(
                                make_shared<EntryRepr>(
                                        make_shared<GetInfoEntry>(
                                                make_shared<FakeScores>(
                                                        make_shared<ServerScore>(
                                                                make_shared<PredefinedOptions>(
                                                                        "host", "host",
                                                                        "port", "777",
                                                                        "pubkey", "pppppppp",
                                                                        "wallet", "666"
                                                                )

                                                        )
                                                )
                                        ),
                                        make_shared<HttpRequest>("")
                                ),
                               R"("host":"host")",
                               R"("port":"777")",
                               R"("invoice":"pppppppp@666")"
                        )
```

**Comment by 0pdd** [comment]
@DronMDF the puzzle [#313](https://github.com/DronMDF/2out/issues/313) is still not solved.

**Comment by 0pdd** [comment]
@DronMDF 3 puzzles [#356](https://github.com/DronMDF/2out/issues/356), [#357](https://github.com/DronMDF/2out/issues/357), [#358](https://github.com/DronMDF/2out/issues/358) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313).

**Comment by 0pdd** [comment]
@DronMDF 4 puzzles [#357](https://github.com/DronMDF/2out/issues/357), [#358](https://github.com/DronMDF/2out/issues/358), [#394](https://github.com/DronMDF/2out/issues/394), [#395](https://github.com/DronMDF/2out/issues/395) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356).

**Comment by 0pdd** [comment]
@DronMDF 3 puzzles [#357](https://github.com/DronMDF/2out/issues/357), [#394](https://github.com/DronMDF/2out/issues/394), [#395](https://github.com/DronMDF/2out/issues/395) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356), [#358](https://github.com/DronMDF/2out/issues/358).

**Comment by 0pdd** [comment]
@DronMDF 5 puzzles [#357](https://github.com/DronMDF/2out/issues/357), [#395](https://github.com/DronMDF/2out/issues/395), [#398](https://github.com/DronMDF/2out/issues/398), [#399](https://github.com/DronMDF/2out/issues/399), [#400](https://github.com/DronMDF/2out/issues/400) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356), [#358](https://github.com/DronMDF/2out/issues/358), [#394](https://github.com/DronMDF/2out/issues/394).

**Comment by 0pdd** [comment]
@DronMDF 4 puzzles [#357](https://github.com/DronMDF/2out/issues/357), [#398](https://github.com/DronMDF/2out/issues/398), [#399](https://github.com/DronMDF/2out/issues/399), [#400](https://github.com/DronMDF/2out/issues/400) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356), [#358](https://github.com/DronMDF/2out/issues/358), [#394](https://github.com/DronMDF/2out/issues/394), [#395](https://github.com/DronMDF/2out/issues/395).

**Comment by 0pdd** [comment]
@DronMDF 3 puzzles [#357](https://github.com/DronMDF/2out/issues/357), [#398](https://github.com/DronMDF/2out/issues/398), [#399](https://github.com/DronMDF/2out/issues/399) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356), [#358](https://github.com/DronMDF/2out/issues/358), [#394](https://github.com/DronMDF/2out/issues/394), [#395](https://github.com/DronMDF/2out/issues/395), [#400](https://github.com/DronMDF/2out/issues/400).

**Comment by 0pdd** [comment]
@DronMDF 2 puzzles [#357](https://github.com/DronMDF/2out/issues/357), [#398](https://github.com/DronMDF/2out/issues/398) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356), [#358](https://github.com/DronMDF/2out/issues/358), [#394](https://github.com/DronMDF/2out/issues/394), [#395](https://github.com/DronMDF/2out/issues/395), [#399](https://github.com/DronMDF/2out/issues/399), [#400](https://github.com/DronMDF/2out/issues/400).

**Comment by 0pdd** [comment]
@DronMDF 2 puzzles [#357](https://github.com/DronMDF/2out/issues/357), [#405](https://github.com/DronMDF/2out/issues/405) are still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356), [#358](https://github.com/DronMDF/2out/issues/358), [#394](https://github.com/DronMDF/2out/issues/394), [#395](https://github.com/DronMDF/2out/issues/395), [#398](https://github.com/DronMDF/2out/issues/398), [#399](https://github.com/DronMDF/2out/issues/399), [#400](https://github.com/DronMDF/2out/issues/400).

**Comment by 0pdd** [comment]
@DronMDF the puzzle [#405](https://github.com/DronMDF/2out/issues/405) is still not solved; solved: [#313](https://github.com/DronMDF/2out/issues/313), [#356](https://github.com/DronMDF/2out/issues/356), [#357](https://github.com/DronMDF/2out/issues/357), [#358](https://github.com/DronMDF/2out/issues/358), [#394](https://github.com/DronMDF/2out/issues/394), [#395](https://github.com/DronMDF/2out/issues/395), [#398](https://github.com/DronMDF/2out/issues/398), [#399](https://github.com/DronMDF/2out/issues/399), [#400](https://github.com/DronMDF/2out/issues/400).
